### ISSUE TITLE
[Bug]: Unable to access history in Nu 0.92

### ISSUE THREAD

**Issue by c4lliope** [atuinsh/atuin#1951]
### What did you expect to happen?

Hello! I applied the changed `atuin.nu`, as recorded on the main branch:
https://github.com/atuinsh/atuin/blob/main/atuin/src/shell/atuin.nu
( @ 5a4dc6b5075ae1281bfc37bca2d9e960cc1de7c1 )

This cleans up the error message I had seen since my nushell upgrade -
though I keep seeing the basic nushell `ctrl-r` handler.
Unsure how my Atuin keybindings became messed up here -
the history is recorded properly when I check using `atuin history last`,
and I see the `pre_prompt` and `pre_execution` hooks are applied as normal.

### What happened?

```nu
~/.build/config> nu -v                                  04/15/2024 04:22:40 AM AM
0.92.1
~/.build/config> atuin -V                               04/15/2024 04:22:43 AM AM
atuin 18.1.0
~/.build/config> $env.config | get -i hooks             04/15/2024 04:22:44 AM AM
╭───────────────────┬───────────────────────────────────────────────────────────╮
│ pre_prompt        │ [list 2 items]                                            │
│ pre_execution     │ [list 2 items]                                            │
│ env_change        │ {record 1 field}                                          │
│ display_output    │ if (term size).columns >= 100 { table -e } else { table } │
│ command_not_found │ <Closure 1341>                                            │
╰───────────────────┴───────────────────────────────────────────────────────────╯
~/.build/config?                                        04/15/2024 04:22:45 AM AM
0: $env.config | get -i hooks
1: atuin -V
2: nu -v
3: atuin doctor
4: nn
5: ls nushell/
6: cd .build/config
7: ch ~/.build/config/nushell/config.nu
8: ch ~/.config/nushell/config.nu
9: atuin --help
Page 1: records 0 - 9  total: 2810
```

### Atuin doctor output

```yaml
Atuin Doctor
Checking for diagnostics


Please include the output below with any bug reports or issues

atuin:
  version: 18.1.0
  sync: null
shell:
  name: nu
  plugins: []
system:
  os: NixOS
  arch: x86_64
  version: '24.05'
  disks:
  - name: /dev/disk/by-uuid/edb81381-1799-41aa-a645-3648cbdffea0
    filesystem: ext4
  - name: /dev/disk/by-uuid/edb81381-1799-41aa-a645-3648cbdffea0
    filesystem: ext4
  - name: /dev/nvme0n1p1
    filesystem: vfat
```
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by c4lliope** [comment]
Also I hopped back and pulled in the upgraded nushell sample config here:
https://github.com/nushell/nushell/blob/main/crates/nu-utils/src/sample_config/default_config.nu

(keeping the lines that I use to load in atuin.sh and my local nu functions).

**Comment by YummyOreo** [comment]
It might be because the keybindings didnt are defined in atuin.nu but instead in the code. To fix this, update atuin to 18.2.0 and run `atuin init nu | save -f ~/.local/share/atuin/init.nu`

**Comment by ccalhoun1999** [comment]
> It might be because the keybindings didnt are defined in atuin.nu but instead `atuin init` adds they keybindings to the code in `atuin.nu`. To fix this, update atuin to `18.2.0` and run `atuin init nu | save -f ~/.local/share/atuin/init.nu`

This fixed the issue for me. I had the issue in `18.1.0`.

**Comment by c4lliope** [comment]
Yep! Same here, 18.2 seems really good on nu.
